### DISCUSSION TITLE
Scrum Prep Milestone 1 -Week 4

### DISCUSSION THREAD

**Discussion by AngelUPR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#93] [Show and tell]
In a sentence or less say what you have been working on during the week

**Comment by Adrianortiz-PR** [comment]
Helped team member to install repo and started to divide tasks for team members

**Comment by MiguelGibo** [comment]
Started learning svelte and downloaded GitHub repo.

**Comment by estefania-2817** [comment]
I finished the svelte tutorial, almost finishing the code for the user interface for the sign up pages. Which includes a page for choosing between a student account or organization account; and a register form page for each of the types of accounts.

**Comment by JeanSanchezFelix** [comment]
Revised the tasks for the Homepage team for the Milestone 1

**Comment by Ingrid1089** [comment]
This week i finished learning Flask and Svelte.

**Comment by Mannyrodd101** [comment]
This week I couldn't do much other than learning some additional things about svelte due to having to study for two exams. Planning on finishing the profile icon by next week.

**Comment by EdzelOrtiz** [comment]
This week I was planning and researching features to add or to start for the Profile Tuner in terms of the skeleton of the webpage, Faculties, Majors, and associations.

**Comment by kevin-d-lopez** [comment]
Finish task that where assigned for milestone 1.
